src/gretel_client/transformers/transformers/secure_fpe.py

```diff
@@ -16,6 +16,8 @@ class SecureFpeConfig(RestoreTransformerConfig):
     radix: int = None
     type: type = None
     secret: str = None
+    mask: str = None
+    mask_fpe_char: chr = '0'
     float_precision: int = None
     aes_mode: Mode = Mode.CBC
```

Comment:
Please add **detailed** doc strings for this transformer config. It's a pretty complex one so we'll want to make sure each param is detailed. Also we can add simple in-line examples to doc strings too.

Example of doing that:

```
Args:
    foo: ...
    bar: ...

Example mask usage::

    sample python code here

```

Reply:
Another question, are `mask` and `float_precision` mutually exclusive? If both are not `None` should we throw a `ValueError` or something?

Reply:
They are not. Float precision only works on values of type float not strings. Mask on the other hand only works on strings... resolving this cleanly does bring up the mask discussion again as at some point I was toying around with the idea to use the mask to specify the precision like "*.???" meaning preserve anything in front of the decimal point and up to 3 digits beyond the decimal point.